hbase-client/src/main/java/org/apache/hadoop/hbase/shaded/protobuf/ProtobufUtil.java

```diff
@@ -2932,12 +2932,17 @@ public static SnapshotType createSnapshotType(SnapshotProtos.SnapshotDescription
     if (snapshotDesc.getCreationTime() != -1L) {
       builder.setCreationTime(snapshotDesc.getCreationTime());
     }
+    if (snapshotDesc.getTtl() != -1L && snapshotDesc.getTtl() < Long.MAX_VALUE / 1000) {
```

Comment:
`snapshotDesc.getTtl() < Long.MAX_VALUE / 1000`, not quite understand here, would you mind explaining it?

Reply:
Hmm, you are converting the time to ms, i get you.

Reply:
TimeUnit has utility functions for this, better than dividing by constants as the intent is clear.